internal/cmd/gocdk/biome.go

```diff
@@ -142,6 +143,30 @@ func biomeAdd(ctx context.Context, pctx *processContext, biome, launcher string)
 			&launchSpecifier{Key: "location", Value: strconv.Quote("${local.gcp_region}")},
 			&launchSpecifier{Key: "service_name", Value: strconv.Quote(serviceName)},
 		)
+	case "ecs":
+		pctx.Logf("")
+		pctx.Logf("To launch on ECS, we need a few pieces of information.")
+		reader := bufio.NewReader(pctx.stdin)
+		region, err := prompt.AWSRegion(reader, pctx.stderr)
+		if err != nil {
+			return xerrors.Errorf("biome add: %w", err)
+		}
+		saveActions = append(saveActions,
+			static.AddLocal(prompt.AWSRegionTfLocalName, region),
+			static.AddProvider("aws"),
+			static.AddProvider("random"),
+			&static.Action{
+				SourcePath:   "/launchers/ecs.tf",
+				DestRelPath:  "main.tf",
```

Comment:
WDYT about creating this as a new file rather than appending it? It seems big enough.